src/django/api/views/v1/opensearch_query_builder/opensearch_query_builder_interface.py

```diff
@@ -0,0 +1,40 @@
+from abc import abstractmethod
+
+
+class OpenSearchQueryBuilderInterface:
```

Comment:
Will we reused it in the future?

Reply:
Most likely, no.

Reply:
> Will we reused it in the future?

It can be used to create a new query builder for other GET endpoints.

Reply:
> It can be used to create a new query builder for other GET endpoints.

Thats possible, but for now I have only once common `OpenSearchQueryBuilder`. I introduced`OpenSearchQueryBuilderInterface` in this PR just for sake of clean architecture and for delelopers who want to quickly check which methods are implemented in query builder.

Reply:
Agreed, good idea that fully aligns with the Builder pattern